protocol-definitions/Map.yaml

```diff
@@ -2965,4 +2965,26 @@ methods:
           since: 2.4
           doc: |
             the function to apply to each entry.
-    response: {}
\ No newline at end of file
+    response: {}
+  - id: 73
+    name: putAllWithMetadata
+    since: 2.7
+    doc: |
+      Put a batch of entries with metadata into an IMap. This is a client side version of WAN entry replication.
+    request:
+      retryable: false
+      partitionIdentifier: any key belongs to target partition
+      params:
+        - name: name
+          type: String
+          nullable: false
+          since: 2.7
```

Comment:
Is this version correct?

Reply:
I checked it. 2.6 is for 5.3. Therefore, 2.7 should be fore 5.4 and it is correct.